api.rb

```diff
@@ -76,6 +76,8 @@ class Api < Roda
   plugin :halt
 
   ASSETS = Assets.new(precompiled: PRODUCTION)
+  # TODO: This should scan the directory
+  PIN_ASSETS = { 'deadbeef' => Assets.new(precompiled: true, pin: 'deadbeef') }.freeze
```

Comment:
we shouldn't serve these assets or render them in opal on the server, let's only support static gzip file

Reply:
I'll get everything else working then try and fix this, there's a question of how to disable the prerender.

Reply:
i think in addition the static gzip, we'll need to generate the game page html

Reply:
yeah, that's why i went down the asset route since it already knew how to generate it.

Reply:
take a look at api.rb#134 and #149 
if you want it to look nice you'll need to do something like 137, else you can try 153